compiler/Makefile

```
@@ -36,9 +34,6 @@ OBELISK ?= obelisk
 # --------------------------------------------------------------------
 .PHONY: all build byte native CIL check check-ci __force__
```

```suggestion
.PHONY: all build byte native CIL check check-ci
```
`__force__` is not needed anymore

---

This one also